src/pcrextend/pcrextend.c

```diff
@@ -65,6 +67,7 @@ static int help(int argc, char *argv[], void *userdata) {
                "     --machine-id        Measure machine ID into PCR 15\n"
                "     --product-id        Measure SMBIOS product ID into NvPCR 'hardware'\n"
                "     --early             Run in early boot mode, without access to /var/\n"
+               "     --event-type=TYPE   Event type identifier to include in the event log\n"
```

Comment:
Just "event type", w/o "indentifier". (The whole *event* is what is included. The *identifier* that is part of the event entry is included too, but that's just incidental, the user cares about the event.)